cmake/projects/Libevent/hunter.cmake

```diff
@@ -0,0 +1,23 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME Libevent
+    VERSION "2.1.8"
+    URL "https://github.com/isaachier/libevent/archive/hunter-2.1.8-p5.tar.gz"
```

Comment:
Please set this release:
* https://github.com/hunter-packages/libevent/releases/tag/v2.1.8-p0